.github/workflows/integration-test.yml

```diff
@@ -17,7 +17,6 @@ on:
       - 'go.mod'
       - 'go.sum'
       - '!**.md'
-      - '!internal/buildscripts/**'
```

Comment:
Just to make sure I understand: Are we confident the integration tests have no relevance to anything within the buildscripts directory? If there's no relevance, do we have any tests that would be impacted by upgrading the jmx metric gatherer version?

Reply:
`!internal/buildscripts/**` actually excludes any file changed in `internal/buildscripts`, like the version file in this PR. So I'm removing this filter so that the [jmxreceiver test](https://github.com/signalfx/splunk-otel-collector/tree/main/tests/receivers/jmxreceiver) runs when this file is changed.

Reply:
Of course, I totally misread this change. Thanks Jeff!